src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml

```diff
@@ -2,17 +2,19 @@
 # (c) Paweł Jędrzejewski
 
 imports:
+    - { resource: "@SyliusUiBundle/Resources/config/app/config.yml" }
+
     - { resource: "@SyliusShopBundle/Resources/config/app/sylius/sylius_mailer.yml" }
     - { resource: "@SyliusShopBundle/Resources/config/grids/account/order.yml" }
     - { resource: "@SyliusShopBundle/Resources/config/grids/product.yml" }
 
 sylius_grid:
     templates:
         action:
-            show: "@SyliusUi/Grid/Action/show.html.twig"
-            pay: "@SyliusShop/Account/Order/Grid/Action/pay.html.twig"
```

Comment:
This renaming is still a BC break, we should leave `pay`, deprecate its template and add `shop_payz to follow the convention.